lib/plugins/aws/provider/awsProvider.js

```diff
@@ -142,7 +142,39 @@ class AwsProvider {
       serverless.configSchemaHandler.defineProvider('aws', {
         function: {
           // TODO: Complete schema, see https://github.com/serverless/serverless/issues/8017
-          properties: { handler: { type: 'string' } },
+          properties: {
+            handler: { type: 'string' },
+            fileSystemConfigs: {
+              description:
+                'Parameters describing connection between a Lambda function and an Amazon EFS file system',
+              type: 'array',
+              minItems: 1,
+              maxItems: 1,
+              items: {
+                type: 'object',
+                properties: {
+                  localMountPath: { type: 'string', pattern: '^/mnt/[a-zA-Z0-9-_.]+$' },
+                  arn: {
+                    oneOf: [
+                      {
+                        $ref: '#/definitions/awsArn',
+                      },
```

Comment:
AWS provides quite specific pattern for this ARN: https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-lambda-function-filesystemconfig.html

Let's maybe the configure it in place with it. It'll ensure better validation

Reply:
Updated :+1: